release_dashboard/test/widgets/conductor_status_test.dart

```diff
@@ -199,5 +184,24 @@ void main() {
       expect(tester.widget<ExpansionPanelList>(find.byType(ExpansionPanelList).first).children[0].isExpanded,
           equals(true));
     });
+
+    testWidgets('Repo Info section displays URLs', (WidgetTester tester) async {
```

Comment:
What about also the case where it says unknown?

Reply:
URL never displays unknown. It always result to a default LUCI url the way that the status is being obtained.